frontend/src/debug.cljs

```diff
@@ -67,7 +68,7 @@
   #{:app.main.data.workspace.notifications/handle-pointer-update
     :app.main.data.workspace.selection/change-hover-state})
 
-(defonce ^:dynamic *debug* (atom #{#_:events}))
+(defonce ^:dynamic *debug* (atom #{#_:events :thumbnails}))
```

Comment:
enabled by default the `:thumbnails` debug flag?

Reply:
ups, mistake. I'll remove it